src/index.html

```diff
@@ -1,13 +1,73 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page">
+    <h1>Product cards</h1>
+    <h2>Some product cards</h2>
+
+    <div class="page__product-card product-card">
+
+      <div class="product-card__wrapper-image-zone">
+        <img
+          class="product-card__image image"
+          src="images/imac.jpeg"
+          alt="images of product card"
+        >
+      </div>
+
+
+      <div class="product-card__wrapper-content">
```

Comment:
one empty line maximum

```suggestion
        >
      </div>

      <div class="product-card__wrapper-content">
```